test/JDBC/input/key_column_usage-vu-prepare.sql

```diff
@@ -0,0 +1,29 @@
+CREATE DATABASE key_column_usage_vu_prepare_db;
+GO
+
+USE key_column_usage_vu_prepare_db;
+GO
+
+CREATE TABLE key_column_usage_vu_prepare_tbl1(arg1 int, arg2 int, primary key(arg1));
```

Comment:
test cases with check constraints and unique constraints should be added

Reply:
Addressed